lambdas/validate-oauth-callback/src/main/java/uk/gov/di/ipv/core/validateoauthcallback/ValidateOAuthCallbackHandler.java

```diff
@@ -242,7 +242,7 @@ private void setIpvSessionCRIAuthorizationCode(
     }
 
     @Tracing
-    private String getPersistedOauthState(CredentialIssuerRequestDto request) {
```

Comment:
Param name to criCallbackRequest.

Reply:
Oops. Good spot that this one wasn't renamed. Using `callbackRequest` as used elsewhere.